### ISSUE TITLE
Replace all individual tests[].run() invocations with run_tests()

### ISSUE THREAD

**Issue by Fbonazzi** [RandomPower/IID_validation#103]
After #101 has been merged, `permutation_tests.run_tests()` can be used to execute any subset of the existing tests.

Replace all direct invocations of permutation_tests.tests[].run() with run_tests(), especially if said invocations need to take care of tests 8 and 9 (already done inside run_tests(), no need to duplicate) or if you need to call them repeatedly over multiple tests.

Example:
```python
if test in [8, 9]:
    Ti.append(permutation_tests.tests[test].run(seq, utils.config.config_data['statistical_analysis']['p_value_stat']))
else:
    Ti.append(permutation_tests.tests[test].run(seq))
```
can probably simply be replaced by something like
```python
Ti.extend(permutation_tests.run_tests(seq, utils.config.config_data['statistical_analysis']['p_value_stat'], [test])
```
